src/utilities/APIHandler/apiHandler.test.js

```diff
@@ -44,6 +44,29 @@ test('APIHandler can handle a mock get call that returns an error', async () =>
     expect(fetchResult).toEqual(error)
 })
 
+test('APIHandler can make a mock post call', async () => {
+    const itemData = {
+        "data": {
+            "data": [
+                { "id": 1 },
+                { "id": 2 },
+                { "id": 3 },
+            ]
+        }
+    };
+
+    const apiHandler = new APIHandler(new MockFetcher(itemData));
```

Comment:
I should have mentioned this back in the other PR... but I noticed that every time we setup this APIHandler in our tests, we build up the MockFetcher with it, is there a specific reason we are mocking the underlying Fetcher and not the APIHandler?

Reply:
I had wanted to keep APIHandler as the sole class that the rest of the application interacts with, which felt important to keep consistent when I do larger integration tests. 
But afterring looking at the kinds of tests I AM writing, it does seem like I could just have 2 Handlers: MockAPIHandler and APIHandler. I'll update the PR to reflect that change